app/scripts/components/RegistrationsPaidPopover/RegistrationsPaidPopover.tsx

```diff
@@ -0,0 +1,9 @@
+export const RegistrationsPaidPopover = (): JSX.Element => {
+  return (
+    <div>
+      <button className="btn btn-default btn-xs">Balance Due</button>{' '}
```

Comment:
I wonder if `&nbsp;` is better than a space like this? Or perhaps better still is to use CSS to separate these instead.

Reply:
This is mostly a prettier artifact. prettier wants to break up the line to keep it from being too long, but since whitespace is trimmed at the end of a line, it has to use `{' '}` to force a space. But I can switch to using CSS to separate the buttons.